### ISSUE TITLE
Adding tooltips for user information

### ISSUE THREAD

**Issue by aborroy** [AlfrescoLabs/alfresco-docker-extension#30]
Help users understand what the first icon on each row means by adding a tooltip when hovering over, consider accessibility and how some people cannot distinguish colours or find the icon too small to get its meaning. 

Same for the info button at the end of each row, tell the users what to expect if they click there or trade it with text; it also does not need to be a primary button, see [here](https://www.figma.com/file/U7pLWfEf6IQKUHLhdateBI/Docker-Extensions-UI-Styleguide?node-id=449%3A12545) how to use different buttons.
